Evaluation.py

```diff
@@ -42,17 +49,30 @@ def __init__(self, name = None, tableFile = None):
         self.nSubstitutions = 0
         self.nStringErrors = 0
         if self.tableFile:
-            row = [ column for column, var in self.tableFormat if column is not None ]
-            print(u'\t'.join(row), self.tableFile)
-
-    tableFormat = [
-        (None,      '"".join(source)'),
-        ('weight',  'weight'),
-        ('symbols', 'nSymbols'),
-        ('ins',     'nInsertions'),
-        ('del',     'nDeletions'),
-        ('sub',     'nSubstitutions'),
-        ('err',     'nStringErrors')]
+            print(self.build_row(True), file=self.tableFile)
+
+    def build_row(self, header, source=tuple(), weight=None, nSymbols=None,
+                  nInsertions=None, nDeletions=None,
+                  nSubstitutions=None, nStringErrors=None):
+        tableFormat = [
+            (None,      "".join(source)),
+            ('weight',  weight),
+            ('symbols', nSymbols),
+            ('ins',     nInsertions),
+            ('del',     nDeletions),
+            ('sub',     nSubstitutions),
+            ('err',     nStringErrors)]
+        if header:
+            row = [
+                unicode(column) for column, var in tableFormat
+                if column is not None
+            ]
+        else:
+            row = [
+                unicode(var) for column, var in tableFormat
+                if column is not None
```

Comment:
@hatzel @langep are you guys sure the if statement should be here?

Reply:
Good catch! In principle you are correct. However this does create a results log of the following form:
```
weight  symbols ins     del     sub     err
test    1       4       0       0       0       0
teXst   1       4       1       0       0       1
```
Which looks confusing to me. The current Python 2 version in this repo does not seem to produce a header at all (due to `print(u'\t'.join(row), self.tableFile)` not using a named parameter).

The original Python-2-only version doesn't appear to have this bug (`print >> self.tableFile, u'\t'.join(row)`). This means the number of content columns didn't originally match the number of header columns either.

I'd personally be in favor of removing the filter altogether. To create a more legible table output:
```
None	weight	symbols	ins	del	sub	err
test	1	4	0	0	0	0
teXst	1	4	1	0	0	1
```
There would also be no reason not to give the `None` column a sensible name. Perhaps 'entry' or 'word'?

So two options:
* Just remove the filter in the non-header branch: confusing output but less changes
* Remove both filters, creating nicer output

I'd be fine with implementing either one.